easybuild/easyconfigs/d/DL_POLY_Classic/DL_POLY_Classic-1.9-intel-2016b.eb

```diff
@@ -0,0 +1,16 @@
+name = 'DL_POLY_Classic'
```

Comment:
No, I mostly created this to verify that the enhanced easyblock is not broken for DL_Poly Classic without PLUMED, doesn't hurt to include that too.